woocommerce/woocommerce-ios
# [Woo POS] [WIP] Show inline message for payment alerts

Closes: #


## Description


Based on Josh's https://github.com/woocommerce/woocommerce-ios/pull/13040

This PR shows that we can split the alert details into 2 different presentation types to be shown as "alert" or "message". It maps to the corresponding type in `CardPresentPaymentsAlertPresenterAdaptor`. There are a few TODO comments but they could potentially be done separately since all changes are affecting POS only.

## Testing information


Prerequisite: the store is eligible for POS and the feature switch is enabled in Menu > Settings > Experimental Features > POS
* Launch app
* Go to Menu > Point of Sale
* Add some item(s) to cart, then tap `Collect payment` --> in the happy path, the connection alerts should be shown as a sheet while payment events are shown inline in the totals view (as a temporary Text view with an orange background)

## Screenshots



https://github.com/woocommerce/woocommerce-ios/assets/1945542/d3bb599f-b8db-47d9-818e-6c958e091bba



---
